lib/reports/utils.ts

```diff
@@ -31,15 +31,14 @@ const sendConfirmationEmail = (to: string): Promise<void> => {
   });
 };
 
-// TODO: Update email content
-const text = 'Your parking permit application has been received';
+const text = 'Your Parking Permit Renewal Application was Successfully Submitted';
```

Comment:
I want to clarify that the `text` parameter above is used as a fallback in case the HTML email cannot be rendered in the email client, so the `text` value should be very similar to, if not the same as the HTML content, just without the styling.

Reply:
Thanks for letting me know! Updated the text content accordingly